postgres.go

```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/go-rel/sql/builder"
 )
 
+// TimestampFormat used by PostgreSQL adapter.
+const TimstampFormat = "2006-01-02 15:04:05.999999999Z07:00:00"
```

Comment:
should we use the same variable name as in sql package?
https://github.com/go-rel/sql/blob/main/util.go#L11

Reply:
I feel we should not specify fixed timezone (`Z07:00:00`) here 🤔

Reply:
how do you mean fixed timezone?

Reply:
`Z07:00:00` is format for timezone (it will not change variable time zone) - see: https://play.golang.org/p/_ytFMrUOXCJ

Reply:
My bad again 🤦 (since jakarta timezine also +7)

Reply:
How about variable naming?

Reply:
oh, you mean typo? 😄

Reply:
Renamed constant to be same as in go-rel/sql and moved formatting to new function that could be reused also externally